torch/_dynamo/variables/builder.py

```diff
@@ -139,6 +139,33 @@ def get_fake_examples(self):
             assert isinstance(
                 self.fake_tensor, torch._subclasses.fake_tensor.FakeTensor
             )
+            # For inplace ops changing the input's shape (unsqueeze_)
+            if (
+                not config.dynamic_shapes
+                and self.fake_tensor.shape != self.example.shape
+            ):
+                converter = torch._subclasses.fake_tensor.FakeTensorConverter()
+                self.fake_tensor = converter.from_real_tensor(
+                    self.fake_tensor.fake_mode, self.example
+                )
+            elif config.dynamic_shapes and self.fake_tensor.dim() != self.example.dim():
```

Comment:
Is it a complete check to make sure there was inplace op on the inputs?

Reply:
This check is updated in [5999b38](https://github.com/pytorch/pytorch/pull/90150/commits/5999b38b1b50fd995665b00b8a9e82f75f920af6)